data/src/main/java/com/example/data/Data.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.data
+
+import android.net.ConnectivityManager
+import android.util.Log
+import com.example.domain.Base
+import com.example.domain.Item
+import com.google.gson.GsonBuilder
+import retrofit2.*
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Data {
+    var repo: MutableMap <String, MutableList<Base>> = mutableMapOf()
+
+    private fun loadData(item: String, callback: (List<Base>) -> (Unit)) {
+        var retrofit: Retrofit = Retrofit.Builder()
```

Comment:
We could move the url to `strings.xml` file.

And also, the retrofit instance is created every time the `loadData` function is called right? We can improve performance if we create this instance just once, and then reuse it